lib/ReactViews/DataCatalog/DataCatalogItem.jsx

```diff
@@ -114,4 +119,8 @@ const DataCatalogItem = createReactClass({
   }
 });
 
-module.exports = DataCatalogItem;
+module.exports = {
+  default: withRouter(DataCatalogItem),
+  DataCatalogItem: withRouter(DataCatalogItem),
```

Comment:
Probably not a huge problem, but if I'm not mistaken calling `withRouter` twice will create two separate but identical functions. Better to avoid that I think.